From 56978f3bec6e799eefa7aeaca803642ca00cd79a Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Mon, 15 Jun 2015 12:28:25 -0400
Subject: [PATCH 780/933] Fix: Incorrect Ubuntu kernel patchlevel

The kernel version used should be the one in Makefile and generated
version.h, not the one tweaked by Ubuntu as "0" in the built package.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 drivers/staging/lttng/instrumentation/events/lttng-module/kmem.h | 4 ++--
 drivers/staging/lttng/lttng-statedump-impl.c                     | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/staging/lttng/instrumentation/events/lttng-module/kmem.h b/drivers/staging/lttng/instrumentation/events/lttng-module/kmem.h
index 6461788..85c116b 100644
--- a/drivers/staging/lttng/instrumentation/events/lttng-module/kmem.h
+++ b/drivers/staging/lttng/instrumentation/events/lttng-module/kmem.h
@@ -203,8 +203,8 @@ LTTNG_TRACEPOINT_EVENT_INSTANCE_MAP(kmem_mm_page, mm_page_pcpu_drain,
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(3,19,2)	\
 	|| LTTNG_KERNEL_RANGE(3,14,36, 3,15,0)		\
 	|| LTTNG_DEBIAN_KERNEL_RANGE(3,16,7,9,0,0, 3,17,0,0,0,0)	\
-	|| LTTNG_UBUNTU_KERNEL_RANGE(3,13,0,50, 3,14,0,0)	\
-	|| LTTNG_UBUNTU_KERNEL_RANGE(3,16,0,38, 3,17,0,0))
+	|| LTTNG_UBUNTU_KERNEL_RANGE(3,13,11,50, 3,14,0,0)	\
+	|| LTTNG_UBUNTU_KERNEL_RANGE(3,16,7,38, 3,17,0,0))
 
 LTTNG_TRACEPOINT_EVENT_MAP(mm_page_alloc_extfrag,
 
diff --git a/drivers/staging/lttng/lttng-statedump-impl.c b/drivers/staging/lttng/lttng-statedump-impl.c
index dddd34d..3dc61b0 100644
--- a/drivers/staging/lttng/lttng-statedump-impl.c
+++ b/drivers/staging/lttng/lttng-statedump-impl.c
@@ -403,7 +403,7 @@ void lttng_statedump_process_ns(struct lttng_session *session,
 	 */
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(3,17,0) || \
 		LTTNG_UBUNTU_KERNEL_RANGE(3,13,11,36, 3,14,0,0) || \
-		LTTNG_UBUNTU_KERNEL_RANGE(3,16,0,11, 3,17,0,0))
+		LTTNG_UBUNTU_KERNEL_RANGE(3,16,1,11, 3,17,0,0))
 	proxy = p->nsproxy;
 #else
 	rcu_read_lock();
@@ -422,7 +422,7 @@ void lttng_statedump_process_ns(struct lttng_session *session,
 	}
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(3,17,0) || \
 		LTTNG_UBUNTU_KERNEL_RANGE(3,13,11,36, 3,14,0,0) || \
-		LTTNG_UBUNTU_KERNEL_RANGE(3,16,0,11, 3,17,0,0))
+		LTTNG_UBUNTU_KERNEL_RANGE(3,16,1,11, 3,17,0,0))
 	/* (nothing) */
 #else
 	rcu_read_unlock();
-- 
2.0.2

